apps/src/code-studio/pd/teacher_application/teacher_application.jsx

```diff
@@ -415,16 +426,16 @@ const TeacherApplication = React.createClass({
                 return (
                   <tr key={i}>
                     <td>
-                      <FormGroup validationState={this.getRequiredValidationState(question)}>
+                      <FormGroup validationState={this.getLikertValidationState(question)}>
                         <ControlLabel>
                           {csBeliefsQuestions[question]}
                         </ControlLabel>
                       </FormGroup>
                     </td>
                     {
-                      _.times(4, (j) => {
+                      _.times(likertAnswers.length, (j) => {
```

Comment:
how about just

```jsx
likertAnswers.map((answer, j) => (
  <td key={j} style={likertSurveyCell}>
    ...
  </td>
));
```

Or better yet, map over `beliefPoll` since those are the values being used.